posts/templates/posts/_post.html

```diff
@@ -41,11 +41,30 @@
             {% endif %}
             <span class="pull-right">
                 {# author actions (edit, delete, ...) #}
+            {% if request.user.pk == user.pk %}
```

Comment:
You should compare with `post.author.kp`, not just `user.pk`, which currently is only present on ProfileView, but will be unset or wrong in PostDetailView.